docs/src/modules/develop/pages/gke.adoc

```diff
@@ -0,0 +1,53 @@
+= Run on GKE
+
+include::ROOT:partial$attributes.adoc[]
+include::ROOT:partial$include.adoc[]
+
+== Cluster creation
+Create a GKE cluster. We recommend at least 6 vCPUs (ie, a node pool of 3 `n1-standard-2` nodes). Also ensure that the current user is a cluster admin.
+
+Detailed instructions for creating the GKE cluster can be found in the Knative documentation, follow all the steps up to (but not including) installing Knative.
```

Comment:
The README had a link through to the Knative docs. Should we have that still? Looks like the GKE install instructions are no longer in the latest version. The referenced doc source is here:

https://github.com/knative/docs/blob/release-0.12/docs/install/Knative-with-GKE.md

Reply:
Or maybe there's something else better to link to?

Reply:
I guess let's add the link for now:

```suggestion
Detailed instructions for creating the GKE cluster can be found in the https://github.com/knative/docs/blob/release-0.12/docs/install/Knative-with-GKE.md[Knative documentation], follow all the steps up to (but not including) installing Knative.
```